### PR TITLE
Animations

### PR DESCRIPTION
**Pull request by leighajarett**
### Description
Animations

Linear ticket and [magic word](https://linear.app/docs/github#link-prs) Fixes DEVR-341 

## Type of change
- [ ] Typo fix 
- [x] New feature 
- [ ] Removed outdated references

### CODE REVIEW

**Comment by pinkeshmars** [comment]
Hey @leighajarett and @PoojaB26 This PR is ready to review now :)

File: docs/ff-concepts/animations/widget_animations.md [hunk 1]
```diff
@@ -0,0 +1,232 @@
...
+import fadeEaseOut from '@site//static/img/animations/animation_gifs/fade-easeout.gif';
+
+Widget animations allow you to add animation effects at the widget level.
+To add an animation to a widget, you'll need to go to the property panel for the widget and select the animations tab.
+
+<img src={animationsOverview}  alt="Animation tab within the property panel is used to add widget animations. Animation trigger specifies when the animation will be run. Widget animations can be triggered when the page (or widget in the case of a delayed load) loaded, or within an action. Animation effect specifies the type of animation to use.Animation curve specifies the function used to interpolate values over time. It controls the speed and style of the animation. Animation properties specifies the properties that can be configured for all widget animations, like whether or not it should loop. Effect-specific properties are properties to specific to the animation effect selected. For example, Fade animations allow you to control the initial opacity and the final opacity. You can customize these values to get the look and feel you desire." />
```
Comment (PoojaB26) [inline docs/ff-concepts/animations/widget_animations.md:30]: In the image: "Effect-specific properties are properties to specific to the animation effect selected" should be grammatically be "Effect-specific properties are properties that are specific to the animation effect selected"

File: docs/ff-concepts/animations/widget_animations.md [hunk 2]
```diff
@@ -0,0 +1,232 @@
...
+| **Flip** | Flip animation rotates an element around its horizontal or vertical axis, creating a mirror effect. It's often used for flipping cards in a UI to reveal hidden information. | <img src={flipGif} class="xsmall-image" alt="Alt text for your GIF" /> | `Flip`: The angle at which the widget is viewed. |
+| **Shimmer** | Creates a "shiny" effect moving across the screen, often used to signify that data or content is in the process of loading or being fetched. | <img src={shimmerGif} class="xsmall-image" alt="Alt text for your GIF" /> | `Color`: The color of the "shiny" line or gradient that sweeps of the widget. A common practice is to use a slightly lighter shade than the content. <br></br> <br></br> `Angle`: Determines the direction of the shimmer effect across the content. 0 degrees for horizontal and 90 for vertical. |
+| **Tint** | Adds a color overlay effect to your content. |<img src={tintGif} class="xsmall-image" alt="Alt text for your GIF" /> | `Color`: Color of the overlay. <br></br> <br></br> `Strength`: Intensity of the tint. |
+
```
Comment (PoojaB26) [inline docs/ff-concepts/animations/widget_animations.md:58]: After this section, we can add a disclaimer for which of these animations dont run on Test mode, I remember shimmer doesnt.

File: docs/ff-concepts/animations/widget_animations.md [hunk 3]
```diff
@@ -0,0 +1,232 @@
...
+| ----- | ----- |
+| **Ease In** | Starts the animation slowly and then accelerates towards the end. It's useful for creating an effect where the motion begins gently and speeds up.| 
+| **Ease In Out** | Starts the animation slowly, accelerates in the middle, and then decelerates towards the end. It's ideal for creating smooth, natural-looking animations that don't have abrupt starts or stops.|
+| **Ease Out** | Begins the animation quickly and then slows down towards the end. It gives the effect of a rapid start that gently comes to a stop.|
+| **Bounce** | Adds a bouncing effect at the end of the animation. The animated object overshoots its final position and then bounces back, mimicking the physical behavior of a bouncing ball.|
+| **Elastic** | Creates an elastic effect where the animation overshoots its target value and oscillates before settling. It's useful for animations that need a springy, elastic feel.|
+| **Linear** | Progresses at a constant speed throughout the animation. It provides a uniform transition from start to end, with no acceleration or deceleration.|
+
+
+## How to trigger an animation on page load
```
Comment (PoojaB26) [inline docs/ff-concepts/animations/widget_animations.md:78]: title may be too big: maybe we just say "Animation on Page Load"

File: docs/ff-concepts/animations/widget_animations.md [hunk 4]
```diff
@@ -0,0 +1,232 @@
...
+        mozAllowFullScreen
+        allowFullScreen
+        allow="clipboard-write">
+    </iframe>
+</div>
+<p></p>
+
+<!-- add aracde -->
+
+## Triggering an animation as part of an action flow
```
Comment (PoojaB26) [inline docs/ff-concepts/animations/widget_animations.md:114]: sam: title too big: could be "Animation on Action Trigger"

File: docs/ff-concepts/animations/widget_animations.md [hunk 5]
```diff
@@ -0,0 +1,232 @@
...
+## Applying multiple animations
+You can apply multiple animations to a single widget. By default, when you add multiple animations, they are executed in a series (one after another) creating staggered animation. However, you can define to run all animations at the same time.
+
+### Run multiple animations simultaneously
+If you want to run multiple animations together for the same amount of time (e.g., slide and scale widget at the same time), enable the **Apply same duration & delay** while adding animation.
+
+
+### Create staggered animation
+A staggered animation is multiple animations executed subsequently. Adding staggered animations can help you create a stunning visual effect.
+
```
Comment (PoojaB26) [inline docs/ff-concepts/animations/widget_animations.md:161]: If not now, but maybe in the future we can add a nice diagram to show how multiple animations are divided on a timeline when its staggered.

**Review by PoojaB26 — APPROVED** [review]
Minor changes requested, once done, we can merge this.

File: docs/ff-concepts/animations/widget_animations.md (hunk 1 above)
Comment (pinkeshmars) [reply docs/ff-concepts/animations/widget_animations.md:30]: Done!

File: docs/ff-concepts/animations/widget_animations.md (hunk 3 above)
Comment (pinkeshmars) [reply docs/ff-concepts/animations/widget_animations.md:78]: Done!

File: docs/ff-concepts/animations/widget_animations.md (hunk 4 above)
Comment (pinkeshmars) [reply docs/ff-concepts/animations/widget_animations.md:114]: Done!

File: docs/ff-concepts/animations/widget_animations.md (hunk 5 above)
Comment (pinkeshmars) [reply docs/ff-concepts/animations/widget_animations.md:161]: Yes, of course.

File: docs/ff-concepts/animations/widget_animations.md (hunk 2 above)
Comment (pinkeshmars) [reply docs/ff-concepts/animations/widget_animations.md:58]: ok, I have checked all animations and only the shimmer one is not working in Test mode. So, I have added a note in its own section.

**Comment by pinkeshmars** [comment]
Hey @PoojaB26 I have addressed all the review comments and I am going to merge this.

**Pull request merged**